src/app/components/info-card/info-card.tsx

```diff
@@ -1,4 +1,5 @@
 import { Box, Button, Flex, FlexProps, Stack, Text } from '@stacks/ui';
+import { InfoCardSelectors } from '@tests/selectors/info-card.selectors';
```

Comment:
I wonder if there is a better way to org shared component selectors so we don't get too many individual selector files (same thought with the shared FeesSelector)? Maybe we could have one file called `shared-component.selectors` and export them all from one file?